runtime/src/main/java/uk/ac/manchester/tornado/runtime/graph/TornadoExecutionContext.java

```diff
@@ -52,10 +52,11 @@
 
 public class TornadoExecutionContext {
 
-    private final String name;
     private final int MAX_TASKS = 128;
-    private final ScheduleMetaData meta;
+    private final int INITIAL_DEVICE_CAPACITY = 16;
```

Comment:
Any justification of the number 16?

Reply:
No, random value. Usually, we have [1-5] devices, depending on the system and number of drivers installed.